CHANGES.md

```diff
@@ -75,6 +75,21 @@ OpenSSL 3.2
 
 ### Changes between 3.2.0 and 3.2.1 [xx XXX xxxx]
 
+ * The function EVP_PKEY_public_check() performs various checks on the RSA
```

Comment:
Could you rewrite that to say when the function is used to check RSA, since you can check other key types with it?